### PR TITLE
[5.0] Removal of deprecated language strings

### PR DESCRIPTION
**Pull request by brianteeman**
This pr replaces several others that were created earlier and had conflicts

code review only

Pull Request for Issue # .

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: api/language/en-GB/joomla.ini [hunk 1]
```diff
@@ -685,8 +679,6 @@ JGLOBAL_VOTES="Votes"
 JGLOBAL_VOTES_ASC="Votes ascending"
 JGLOBAL_VOTES_DESC="Votes descending"
 JGLOBAL_WARNCOOKIES="Warning! Cookies must be enabled to access the Administrator Backend."
-; @deprecated 5.0 will be removed in 6.0
```
Comment (Quy) [inline api/language/en-GB/joomla.ini]: Not yet.

Comment (HLeithner) [reply api/language/en-GB/joomla.ini]: @brianteeman can you please revert this

File: administrator/components/com_admin/script.php [hunk 2]
```diff
@@ -2045,6 +2045,10 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
             '/media/com_contenthistory/js/admin-history-versions.min.js.gz',
             // From 5.0.0-beta1 to 5.0.0-beta2
             '/libraries/src/Cache/Storage/WincacheStorage.php',
+            '/administrator/language/en-GB/plg_twofactorauth_totp.ini',
+            '/administrator/language/en-GB/plg_twofactorauth_totp.sys.ini',
+            '/administrator/language/en-GB/plg_twofactorauth_yubikey.ini',
+            '/administrator/language/en-GB/plg_twofactorauth_yubikey.sys.ini',
```
Comment (heelc29) [inline administrator/components/com_admin/script.php]: alphasort https://github.com/joomla/joomla-cms/pull/41721#discussion_r1323301813

**Comment by heelc29** [comment]
I have tested this item :white_check_mark: successfully on 9098fa019c2b1636e2e5b2585cc403b62447a70e<br /><br />only alphasort in script missing<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41717">issues.joomla.org/tracker/joomla-cms/41717</a>.</sub>

**Comment by tecpromotion** [comment]
I have tested this item :white_check_mark: successfully on 9098fa019c2b1636e2e5b2585cc403b62447a70e<br /><br />Thanks @brianteeman <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41717">issues.joomla.org/tracker/joomla-cms/41717</a>.</sub>

File: administrator/components/com_admin/script.php (hunk 2 above)
Comment (heelc29) [reply administrator/components/com_admin/script.php]: Now the entries are double there (alpha4-beta1 and beta1-beta2)

**Pull request merged**